tsconfig.json

```diff
@@ -26,6 +26,7 @@
     "core/test/fixtures/traces/lcp-m78.devtools.log.json",
     "core/audits/byte-efficiency/polyfill-graph-data.json",
     "shared/localization/locales/en-US.json",
+    "shared/statistics.js",
```

Comment:
This line shouldn't be necessary since `./shared/` is in the references section. Do you get any errors in `yarn type-check` if this line is removed?